.env.example

```diff
@@ -0,0 +1,5 @@
+# Example .env file. Do 'cp .env.example .env' and insert appropriate data.
```

Comment:
isn't it `Do 'cp .env.example .env.ENVIRONMENT'` ?

Reply:
The file '.env' is used for local development/debug version.